python/lsst/atmospec/spectraction.py

```diff
@@ -500,6 +523,7 @@ def run(self, exp, xpos, ypos, target, doFitAtmosphere, doFitAtmosphereOnSpectro
 
         # Save the spectrum
         self._ensureFitsHeader(spectrum)  # SIMPLE is missing by default
+        self._ensureFitsHeader(image)  # SIMPLE is missing by default  # TODO: debug this
```

Comment:
Any reason why you added the `TODO: debug this` comment here?

Reply:
The reason is that `atmospec` has never returned the spectractor image object in the collections, I don't know why. First I suspected the header but I am not sure

Reply:
I don't really know what you mean, but this probably isn't the best place to discuss!